### ISSUE TITLE
Language Server reports false positives in Native Notebook

### ISSUE THREAD

**Issue by DonJayamanne** [microsoft/vscode-jupyter#4718]
* Install VSCode Insiders
* Install Jupyter Extension
* Install Pylance
* Select `Python 3.6` as workspace interpreter
* Select a different interrpeter as Kernel in Notebook (e.g. Python 3.7 Conda)
* Install pandas into the kernel interpreter
* Write code in cell and import pandas

Pylance will report errors about missing packages (pandas is reported as not being installed).
This is a false positive, the user has it installed in the python environment used by the kernel.

The problem is pylance doesn't know that the document is using a different interpreter.

This will eventually get even more complicated:
* Assume we import a python module from the notebook
* I now go to deffinition & land in the python file
* Now what Interpreter should the langauge server use when checking pacakges/imports for this file?
Basically it depends on the context, but does the LS know this?

**Comment by rchiodo** [comment]
Dupe of #6333
